experimental/Schemes/src/IdealSheaves.jl

```diff
@@ -1621,6 +1621,16 @@ function produce_object(F::PrimeIdealSheafFromChart, U2::AbsAffineScheme)
     f, g = gluing_morphisms(glue)
     if glue isa SimpleGluing || (glue isa LazyGluing && first(gluing_domains(glue)) isa PrincipalOpenSubset)
       complement_equation(codomain(g)) in F(W) && return ideal(OO(U2), one(OO(U2))) # We know the ideal is prime. No need to saturate!
+      pb_f = pullback(f)::AbsLocalizedRingHom
+      pb_f_res = restricted_map(pb_f)
+      @assert domain(pb_f_res) === ambient_coordinate_ring(V2)
+      Q = preimage(pb_f_res, F(domain(f)))
+      rest = OOX(V2, U2)
+      result = ideal(OO(U2), rest.(gens(Q)))
+      @hassert :IdealSheaves 1 !isone(Q)
+      @hassert :IdealSheaves 1 is_prime(result)
+      set_attribute!(result, :is_prime=>true)
+      return result
```

Comment:
Please either remove the old code or add an algorithm option if you think that it still performs better in some cases.